test/test_playbooks/tasks/composite_content_view.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: "Create/update katello content view"
+  vars:
+    - organization_name: "Test Organization"
+    - composite_content_view_name: "ccv1"
+    - content_view_name: "cv1"
+    - content_view_version: "1.0"
+  katello_content_view:
```

Comment:
Shouldn't this be the shiny new `..._composite_...` module?

Reply:
d'oh